docker/Dockerfile.cpu

```diff
@@ -0,0 +1,67 @@
+FROM ubuntu:16.04
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+         build-essential \
+         git \
+         curl \
+         ca-certificates \
+         libjpeg-dev \
+         libpng-dev \
+         libglib2.0-0 \
+         libsm6 \
+         libxrender-dev \
+         libxext6 \
+         cmake \
+         gcc \
+         protobuf-compiler \
+         libgtk2.0-dev && \
+     rm -rf /var/lib/apt/lists/*
+
+# Install PyTorch
+RUN curl -o ~/miniconda.sh -LO  https://repo.continuum.io/miniconda/Miniconda3-py37_4.8.3-Linux-x86_64.sh  && \
```

Comment:
why conda? Is it possible to use pip? Is there any difference in further usage?

Reply:
No difference in usage, but maybe it's no really need. Ok, i will change it to pip.